logback-classic/src/test/java/ch/qos/logback/classic/pattern/ConverterTest.java

```diff
@@ -398,4 +402,23 @@ public void contextProperty() {
     String result = converter.convert(event);
     assertEquals("v", result);
   }
+
+  @Test
+  @Ignore
+  public void performanceTest() {
```

Comment:
This is no unit test, I just use junit to execute my performance comparison. If you feel happy I can delete the code or create a main method. The first solution won't be good because then you aren't able to check my performance improvements compared to the current state.